code/modules/mob/living/silicon/ai/freelook/chunk.dm

```diff
@@ -84,11 +103,8 @@
 	var/list/newly_obscured_turfs = visibleTurfs - updated_visible_turfs
 
 	for(var/mob/camera/ai_eye/client_eye as anything in seenby)
-		var/client/client = client_eye.ai?.client || client_eye.client
-		if(!client)
-			continue
-
-		client.images -= active_static_images
+		for(var/client/each_client as anything in client_eye.eye_users)
```

Comment:
You really shouldn't be disabling BYOND safety when it's not certain and not in a hot path

Reply:
The same as the another: If that happens, this should throw error. The list should not have anything wrong.